Makefile

```diff
@@ -8,18 +8,19 @@ GOBIN := $(GOBASE)/dist
 GOENVVARS := GOBIN=$(GOBIN) CGO_ENABLED=0 GOOS=linux GOARCH=amd64
 GOBINARY := zkevm-node
 GOCMD := $(GOBASE)/cmd
+$(eval BUILD_COMMIT:=$(shell git rev-parse HEAD))
```

Comment:
@Psykepro do know if we need the full commit hash? Otherwise we could use `--short` flag.

Reply:
@KonradIT what do you think do we need the short or the full?

Reply:
I've updated it to use the short for now.